src/bentoml/container.py

```diff
@@ -389,10 +387,6 @@ def build(
     """
     from ._internal.container import determine_container_tag
 
-    # Run healthcheck
```

Comment:
Is this useful for users using the Python API for containerizing Bento?

Reply:
I think I demonstrated how to use these API in the triton examples.

But I don't think build should call health here.